features/add_ressource_to_item.feature

```diff
@@ -0,0 +1,19 @@
+#language: fr
+
+Fonctionnalité: Ajouter une ressource en tant que fichier 
+
+
+Scénario: Ajouter une ressource en tant que fichier 
+
+Soit "AXN 009" l'item affiché
+Et l'utilisateur est connecté
+Quand l'utilisateur dépose "favicon.ico" comme ressource
+Alors la ressource "favicon.ico" est ajoutée
+
+
+Scénario: Consulter une ressource en tant que fichier 
+
+Soit "AXN 009" l'item affiché
+Et la ressource "favicon.ico" existe comme ressource
```

Comment:
This is considered as a bad practice  to make a test depend on the success of a previous test.
Either find a way to include the attachment in the test data. Or just remove this scenario (and the corresponding steps).